packages/screens/LaunchpadERC20/LaunchpadERC20Screen.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+
+import { LaunchpadERC20Banner } from "./component/LaunchpadERC20Banner";
+
+import { BrandText } from "@/components/BrandText";
+import { OmniLink } from "@/components/OmniLink";
+import { ScreenContainer } from "@/components/ScreenContainer";
+import {
+  LargeBoxButton,
+  LargeBoxButtonProps,
+} from "@/components/buttons/LargeBoxButton";
+import { SpacerColumn, SpacerRow } from "@/components/spacer";
+import { ScreenFC } from "@/utils/navigation";
+import { neutral77 } from "@/utils/style/colors";
+import { fontSemibold14, fontSemibold28 } from "@/utils/style/fonts";
+
+const BUTTONS: LargeBoxButtonProps[] = [
+  {
+    title: "Tokens",
+    description: "Create your own ERC20 token with custom parameters.",
+    buttonTitle: "Open",
+  },
+  {
+    title: "Airdrops",
+    description:
+      "Create airdrops to engage with your community and distribute your tokens.",
+    buttonTitle: "Open",
+  },
+  {
+    title: "Sales",
+    description:
+      "Create private or public sales to distribute your tokens and raise funds.",
+    buttonTitle: "Open",
+  },
+];
+
+export const LaunchpadERC20Screen: ScreenFC<"LaunchpadERC20"> = () => {
+  return (
+    <ScreenContainer>
+      <LaunchpadERC20Banner />
+      <SpacerColumn size={2} />
```

Comment:
Just a small hint, we can add an empty line before and after <Spacer... /> to have a better readability